control-plane/build-support/scripts/consul-version.sh

```diff
@@ -4,5 +4,4 @@
 FILE=$1
 VERSION=$(yq .global.image $FILE)
 
-# echo full string "hashicorp/consul:1.15.1" | remove first and last characters | cut everything before ':'
-echo "${VERSION}" | sed 's/^.//;s/.$//' | cut -d ':' -f2-
+echo "${VERSION}"
```

Comment:
No need to cut a bunch of pieces out as the pipelines will use the full docker image. 

(previously I was trying to hack the docker image together)